radicle-cli/src/commands/issue.rs

```diff
@@ -171,13 +196,24 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     let repo = storage.repository(id)?;
     let mut issues = Issues::open(*signer.public_key(), &repo)?;
 
-    match options.op {
+    let op = match options.op {
+        Operation::Default => Operation::List {
+            assignee: Some(*profile.id()),
+        },
+        _ => options.op,
+    };
+
+    match op {
         Operation::Create {
             title: Some(title),
             description: Some(description),
         } => {
             issues.create(title, description, &[], &signer)?;
         }
+        Operation::Show { id } => {
+            let issue = issues.get(&id)?.context("No issue with such ID exists")?;
```

Comment:
Reads a bit better to me
```suggestion
            let issue = issues.get(&id)?.context("No issue for the given ID exists")?;
```

Reply:
:+1: